### PR TITLE
Update Styling and Variables (Mostly dashboard)

### PR DESCRIPTION
**Pull request by tayvano**
### Things That Changed

1. New variables, lots of finding and replacing
2. Changed order of the ActionTiles on dashboard after talking with Jordan (send, swap, request, buy, hw)
3. Made the "add account" button look the same and be styled the same across Account panel and Address book panel
4. Added some tooltips to settings; translated some random strings that I found
5. **Dashboard spacing updates.** This will now allow the two top columns lower most item to stretch-to-fill so that their bottoms line up, regardless of if the token panel has a lot of tokens, none, or account panel has a lot of accounts or none. The bulk of the work done in this commit (57b6647) was about getting the margins and paddings to be consistent across the dashboard and settings pages. I suspect there may be some weirdness on mobile or tablet sizes, but I'll fix those later as they were already funky anyways. Not a huge deal.
6. Slightly modified **layout for the token details stuff** and fixed the weird padding / scrollbar that was happening.
7. In the token details panel, I added padding around the clickable areas so that the clickable area would be bigger. This should aid with some of the really small SVGs as they essentially have an invisible clickable area around them now. This increased it from ~20px->~50px (~50px is considered accessible)

### Steps to Test

**UI**
1. Look at the dashboard. Resize the screen. Note anything that looks funky
2. Scan for tokens. Note anything that looks funky.
3. Go to settings. Note anything that looks funky.

**UX**
1. Take a look at a couple token details. It should be easier to read and consume the info, and be less tall.
2. Give feedback on new order of the action tiles (top left 5 buttons on the dashboard) (send, swap, request, buy, hw)

**Technical**
1. If it runs, it should be good. This really was mostly finding and replacing strings with variables and CSS work.
2. Note any errors or warnings. 
3. Test yarn start and yarn build (or whatever it is)
4. Teach taylor whatever she needs to know about linting errors because I'm still force pushing (i'm so sorry)

### Note to future frustrated person
- If something is wonky from this PR, check `57b66479a31d144940e8a8957b8007d7811281b6` and `60968238ab4c9b3be7a4981694a6b26ccb0fc62b` and `22089c291f5beb31ef9cc06d941ed50e41674a6e` first as the issue is far more likely to be in one of those than the others.
- I tried to split the commits up so that if we have to blame something or revert something, it's okay.
- I'm so sorry. I just wanted to make variables. I didn't mean to cause you such pain. 😭

### Other Notes (posted in Slack originally)

I fixed some weird padding and now you have motherfucking variables because, honestly, how do you even live. 😂

Here's the scale:
XS
SM
BASE
MD
LG
XL
XXL

`${FONT_SIZE.BASE}` is 16px and `${SPACING.BASE}` is 20px

if its too big, go down to SM or XS. if its too little go up to MD, LG, XL

The same works concept also works for your shades of gray, just slightly different language:

```
  WHITE: ‘#fff’,
  GREY_LIGHTEST: ‘#F7F7F7’,
  GREY_LIGHTER: ‘#E5ECF3’,
  GREY_LIGHT: ‘#D8D8D8’,
  GREY: ‘#B7BFC6’,
  GREY_DARK: ‘#7B8695’,
  GREY_DARKER: ‘#7B8695’,
  GREY_DARKEST: ‘#282D32’,
  BLACK: ‘#000’,
```

So, you can just put gray grey and then go darker or lighter if its not right.

Breakpoints haven’t been updated yet (BREAK_POINTS.SCREEN_XS) but that’s just a tedious thing if you guys like this system.

### Sreenshots!

2020-02-06 at 11 41 01 PM
2020-02-06 at 11 34 36 PM
2020-02-06 at 11 34 15 PM
2020-02-06 at 11 34 07 PM
2020-02-06 at 11 33 52 PM
2020-02-06 at 11 33 47 PM
2020-02-06 at 11 33 43 PM
2020-02-06 at 11 33 35 PM
2020-02-06 at 11 33 25 PM
2020-02-06 at 11 31 56 PM

### CODE REVIEW

File: common/v2/components/AccountList.tsx [hunk 1]
```diff
@@ -159,7 +169,9 @@ export default function AccountList(props: AccountListProps) {
     return (
       <AccountListFooterWrapper>
         <RouterLink to={ROUTE_PATHS.ADD_ACCOUNT.path}>
-          <Typography>{`+ ${translateRaw('ACCOUNT_LIST_TABLE_ADD_ACCOUNT')}`}</Typography>
+          <AddAccountButton basic={true}>{`+ ${translateRaw(
```
Comment (vilsbole) [inline common/v2/components/AccountList.tsx]: what is the `basic` attr for?

File: common/v2/components/Tabs.tsx [hunk 2]
```diff
@@ -24,16 +24,15 @@ interface TabWrapperProps {
 const TabWrapper = styled.div<TabWrapperProps>`
   ${props =>
     props.selected &&
-    `background-color: ${BRIGHT_SKY_BLUE};
-     color: ${WHITE};`}
-  padding: 12px 20px;
+    `background-color: ${BLUE_BRIGHT};
+     color: ${WHITE};`} padding: 12px 20px;
```
Comment (vilsbole) [inline common/v2/components/Tabs.tsx]: missing new-line

File: common/v2/features/Layout/Layout.tsx [hunk 3]
```diff
@@ -62,7 +62,7 @@ const SContainer = styled('div')`
       @media (min-width: ${BREAK_POINTS.SCREEN_SM}) {
         align-self: center;
       }
-    `}
+    `};
```
Comment (vilsbole) [inline common/v2/features/Layout/Layout.tsx]: this may be a linter mistake

Comment (tayvano) [reply common/v2/features/Layout/Layout.tsx]: I think it is—I didn't add this in.

File: common/v2/components/Tabs.tsx (hunk 2 above)
Comment (tayvano) [reply common/v2/components/Tabs.tsx]: The linter thing did this as well. I'll ping you on Slack to see if that's a Taylor+her linter thing or a global linter thing?

File: common/v2/components/AccountList.tsx (hunk 1 above)
Comment (tayvano) [reply common/v2/components/AccountList.tsx]: I copied it from the Address Book component, as they were implemented differently but are essentially the same (add account to account table vs add address to address book table). I figured a button was more appropriate than type, given the context of the user clicking `+ Add Account`. I think the `basic` may produce an unstyled button (no blue background, etc.) as I didn't have to override any styling.

**Comment by tayvano** [comment]
I'm going to NOT go down a rabbithole and ONLY fix mobile styling (per michael's comment) that I broke and add it to this PR.

**Pull request merged**